tock/hours/templatetags/has_submitted_timesheet.py

```diff
@@ -6,9 +6,11 @@
 
 @register.filter(name='has_submitted_timesheet')
 def has_submitted_timesheet(user, reporting_period):
-  if not Timecard.objects.filter(reporting_period=reporting_period,
-                                 time_spent__isnull=False,
-                                 user=user).exists():
-    return False
-  else:
-    return True
+    if not Timecard.objects.filter(
```

Comment:
This can be simplified:
```python
return Timecard.objects.filter(
    reporting_period=reporting_period,
    time_spent__is_null=False,
    user=user,
).exists()
```